Content.Client/Borgs/LawsMenu.xaml.cs

```diff
@@ -0,0 +1,59 @@
+using Content.Client.UserInterface.Controls;
+using Content.Shared.Borgs;
+using Content.Client.Stylesheets;
+using Robust.Client.AutoGenerated;
+using Robust.Client.Graphics;
+using Robust.Client.UserInterface.XAML;
+using Robust.Client.UserInterface.Controls;
+
+namespace Content.Client.Borgs
+{
+    [GenerateTypedNameReferences]
+    public sealed partial class LawsMenu : FancyWindow
+    {
+        [Dependency] private readonly IEntityManager _entityManager = default!;
+        public LawsBoundUserInterface Owner { get; }
+
+        public LawsMenu(LawsBoundUserInterface owner)
+        {
+            RobustXamlLoader.Load(this);
+            IoCManager.InjectDependencies(this);
+            Owner = owner;
+            UpdateLaws();
+
+            if (_entityManager.TryGetComponent<LawsComponent>(owner.Machine, out var laws))
+                StateLawsButton.Visible = laws.CanState;
```

Comment:
@Jezithyr as I'm not sure what the new standards for UI are but I do not think this is allowed.